ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -128,6 +128,9 @@ std::vector<hardware_interface::StateInterface> URPositionHardwareInterface::exp
         info_.joints[i].name, hardware_interface::HW_IF_EFFORT, &urcl_joint_efforts_[i]));
   }
 
+  state_interfaces.emplace_back(
```

Comment:
Should be added here with parameters https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/blob/develop/ur_ros2_control_demos/urdf/ur.ros2_control.xacro 
when generic `gpio` tag is introduced in ros2_control. Can wait, hardcoded like this is fine for now.

Reply:
You can add this now already using `joint` tag.